lib/needs_exporter.rb

```diff
@@ -5,42 +5,47 @@
 class NeedsExporter
   def initialize
     @needs = Need.all
-    @api_client = GdsApi::PublishingApiV2.new(Plek.find('publishing-api'))
+    @api_client = GdsApi::PublishingApiV2.new(Plek.find('publishing-api'), timeout: 30)
     @slugs = Set.new
   end
 
   def run
-    @needs.each_with_index do |need, index|
-      export(need, index)
+    count = @needs.count
+    @needs.order_by(:_id.asc).each_with_index do |need, index|
+      export(need, index, count)
     end
   end
 
 private
 
-  def export(need, index)
+  def export(need, index, count)
     slug = generate_slug(need)
     need_revision_groups = need_revision_groups(need.revisions)
-    snapshots = need_revision_groups.map { |nrg| present_need_revision_group(nrg, slug)}
+    snapshots = need_revision_groups.map { |nrg| present_need_revision_group(need.id, nrg, slug)}
     compute_superseded_needs(snapshots)
-    @api_client.import(need.content_id, snapshots)
+    @api_client.import(need.content_id, "en", snapshots)
     links = present_links(need)
     @api_client.patch_links(need.content_id, links: links)
-    p "#{index}/#{@needs.count}"
-    p "exported #{slug}"
+
+    padding = Math.log10(count).ceil
+    puts format("%#{padding}d/#{count} exported #{slug}", index + 1)
   end
 
-  def present_need_revision_group(need_revision_group, slug)
+  def present_need_revision_group(need_id, need_revision_group, slug)
```

Comment:
Instead of doing this, could you get the need_id from the need_revision directly? In the console, NeedRevision.last.need_id gives me the need_id back.